### PR TITLE
Apollo endpoint usability update

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
* Removes `BASE_URL` from the environment variables file
* Fetch `prefect_ui_settings` static JSON from `/` instead of the base url
* Add a reset method to the GraphQL endpoint setter input, which will clear the `localStorage` `server_url` key and reset default URL to whatever was configured either from the environment or the settings.json

Should make deploying to a remote server with a non-static domain easier

### CODE REVIEW

File: src/pages/Home/StartPrefectServer-Section.vue [hunk 1]
```diff
@@ -114,14 +124,32 @@ export default {
...
+            <v-tooltip top>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  color="blue-grey lighten-1"
+                  icon
+                  small
+                  @click="_resetUrl"
+                  v-on="on"
+                >
+                  <v-fade-transition mode="out-in">
```
Comment (zhen0) [inline src/pages/Home/StartPrefectServer-Section.vue:136]: I find the mix of indicator and action button a bit confusing here.  Could we keep the indicator as is and add a reset button next to the connect button?  (Or something similar.)

**Review by zhen0 — CHANGES_REQUESTED** [review]
I like the addition of the rest but one suggestion/request @znicholasbrown

**Comment by znicholasbrown** [comment]
Good call @zhen0 - that's been updated:
Screen Shot 2020-09-23 at 11 24 23 AM

**Review by zhen0 — APPROVED** [review]
Thanks for the changes @znicholasbrown

**Pull request merged**